week1/index.js

```diff
@@ -1,38 +1,27 @@
-import HTTP from 'http'
-import Path from 'path'
 
-import sendIndexHTML from './responses/sendIndexHTML'
-import sendPage2HTML from './responses/sendPage2HTML'
-import sendStylesCSS from './responses/sendStylesCSS'
-import sendText from './responses/sendText'
+const http = require('http');
+const port = 3000
+var state = 10
 
-const server = HTTP.createServer((request, response) => {
-	console.log(request.method, request.url)
+let server = http.createServer( (request, response) => {
```

Comment:
This is good, but doesn't quite cover all the things we talked about in class. If you have time, I'd like to see you implement setting a Content-Type header and returning some HTML content, interpolated with the 'state' value, instead of just returning `String(state)` for every page.